src/komodo_nSPV_fullnode.h

```diff
@@ -688,6 +688,7 @@ int32_t NSPV_remoterpc(struct NSPV_remoterpcresp *ptr,char *json,int n)
         {
             rpc_result = JSONRPCReplyObj(result, NullUniValue, jreq.id);
             response=rpc_result.write();
+            ptr->json = (char*)malloc(response.size());
```

Comment:
I believe it possible that response.size() could be a large number. Should there be checks to assure that the malloc here (and below) did not return nullptr?

Reply:
definitely should be checked, thank you